jobs/client-regeneration/client_regeneration/main.py

```diff
@@ -379,23 +492,36 @@ def run_simulation(
     column_list: list,
     end_date: str,
     lookback: int,
+    action: str,
```

Comment:
👍

Reply:
do we want to allow for more than one action? esp since the yearly and usage histories are dependent on the clients daily job